### ISSUE TITLE
marker extract does noet work on 4.2.0

### ISSUE THREAD

**Issue by lemonCMS** [biesbjerg/ngx-translate-extract#145]
Hello,

Thanks so much for this package.

For a existing project we needed translations, and ```ngx-translate``` with ```ngx-translate-extract(-marker)``` should be great for this.

All the text in the templates is getting extracted correct.
But the text in the components with the marker function is not on version 4.2.0.

For a test a installed version ```3.0.5``` of ```@biesbjerg/ngx-translate-extract``` and run the extract command with the ```-m marker``` option and it worked, the text in the marker function is extracted. back to version 4.2.0 no text from the components is getting extracted.

Example of the most basic component that works with 3.0.5 but not in 4.2.0

**4.2.0 does not work**
```sh
ngx-translate-extract -i src/_test/ -o test.json -f json
```
**3.0.5 works**
```sh
ngx-translate-extract -i src/_test/ -o test.json -m marker -f json
```

```
import {Component} from '@angular/core';
import {marker} from '@biesbjerg/ngx-translate-extract-marker';

@Component({
  selector: 'app-test',
  templateUrl: './test.component.html',
  styleUrls: ['./test.component.scss']
})
export class TestComponent {
  constructor() {
    const text = marker('TEST_MARKER');
    console.log(text);
  }
}
```

**Comment by smurfy** [comment]
Having same problem

**Comment by jochenjonc** [comment]
Works using the new marker function, see https://github.com/biesbjerg/ngx-translate-extract/blob/master/README.md#marker-function

**Comment by smurfy** [comment]
@jochenjonc well based on @lemonCMS snipped he uses exactly that.
I use the aliased marker function.

I tried again  just now and interestingly it somewhat works, but no longer in html files.
I have some dynamic translations and use html comments with markers to make them extract.
That worked in the past but it seems the marker parser has been changed.

Here is what works in 3.0.5 and not in 4.2.0 for me:

````
<!--_('static.foo.title');-->
<!--_('static.bar.title');-->
<!--_('static.something.title');-->
{{ ('static.' + type + '.title') | translate }}
````
With 3.0.5 it successfully extracts `static.bar.title`, `static.foo.title` and `static.something.title`
4.2.0 will not extract those. Its quite possible that there is an alternative to my marker use. I'm happy to change my code.
